app/helpers/container_image_helper/textual_summary.rb

```diff
@@ -61,21 +57,8 @@ def textual_product_name
     end
 
     def textual_compliance_history
-      h = {:label => _("History")}
-      if @record.number_of(:compliances) == 0
-        h[:value] = _("Not Available")
-      else
-        h[:image] = "compliance"
-        h[:value] = _("Available")
-        h[:title] = _("Show Compliance History of this Container Image (Last 10 Checks)")
-        h[:explorer] = true
-        h[:link] = url_for(
-          :controller => controller.controller_name,
-          :action     => 'show',
-          :id         => @record,
-          :display    => 'compliance_history')
-      end
-      h
+      super(:title    => _("Show Compliance History of this Container Image (Last 10 Checks)"),
+            :explorer => true)
```

Comment:
BTW I don't want to change any existing behavior in this darga/yes PR, just refactor 1:1.